src/mobile/src/ui/components/SeedVaultExportComponent.js

```diff
@@ -193,14 +194,22 @@ class SeedVaultExportComponent extends Component {
     onExportSuccess() {
         const { t } = this.props;
         if (isAndroid) {
-            this.props.generateAlert('success', t('exportSuccess'), t('exportSuccessExplanation'));
-            return this.props.goBack();
+            this.props.goBack();
+            return timer.setTimeout(
+                'timeout',
+                () => this.props.generateAlert('success', t('exportSuccess'), t('exportSuccessExplanation')),
```

Comment:
What is the reason for rendering alert behind a timeout here?

Reply:
To await navigation action